.github/workflows/linters.yaml

```
@@ -36,4 +18,24 @@ jobs:
+
+      - name: Create annotation with file diff
+        run: |
+          echo "::error::Black found code style issues. See details below:"
+          git diff | tee diff_output.txt
+          while IFS= read -r line; do
+            echo "::warning file=$line::Black reformat suggestion"
+          done < <(git diff --name-only)
+
+      # make sure it exits properly on error
+      - name: Run black check
+        run: git stash && black --check .
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)